addons/web_unsplash/static/src/scss/unsplash.scss

```diff
@@ -1,46 +1,30 @@
-.unsplash_img_container {
-    $o-border-image: 2px;
-    .o_unsplash_img_cell {
-        .o_image {
-            cursor: pointer;
-            transition: opacity 400ms;
-
-            &.o_webimage {
-                width: 100%;
-                height: auto;
-                background-image: none;
-
-                img {
-                    max-height: 150px;
-                    margin: 0 auto;
-                    border: $o-border-image solid transparent;
-                }
-            }
-
-        }
-        &.o_selected .o_image img {
-            border-color: $o-brand-primary;
-            box-shadow: 0px 0px 2px 2px $o-brand-secondary;
-        }
-        .author_name {
-            font-size: 11px;
+.o_unsplash_attachment_cell {
+    cursor: pointer;
+    &:hover {
+        .o_we_attachment_highlight {
+            border-color: $card-border-color;
+            box-shadow: 0px 0px 2px 2px $card-border-color;
         }
     }
 
-    .not-found {
-        font-size: 25px;
-        padding: 30px 0;
+    .author_name {
+        font-size: 11px;
     }
+}
+
+.not-found {
```

Comment:
Is this one of our classes ? If so, can it be renamed especially since it is used without context

Reply:
looks like it. Actually I removed this class from the xml completely because the images are much bigger now, no need to change the font-size anymore so I'll just remove this rule entirely.